cinema/views.py

```diff
@@ -21,24 +26,32 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(generics.ListCreateAPIView):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
add TokenAuthentication to the default authentication_classes in settings.py
add IsAdminOrIfAuthenticatedReadOnlyt o the default permission_classes in settings.py
to not duplicate these classes in many places
remove redundant lines of code after

Reply:
But if i do it, the original 3 tests will fall. They raise errors but with another code